velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -837,11 +837,14 @@ void ExpressionFuzzer::logStats() {
   LOG(INFO)
       << "Format: functionName numTimesSelected proportionOfTimesSelected "
          "numProcessedRows";
-  for (int i = entries.size() - 1; i >= entries.size() - maxEntriesLimit; i--) {
-    LOG(INFO) << entries[i].first << " " << entries[i].second.numTimesSelected
-              << " " << std::fixed << std::setprecision(2)
-              << (entries[i].second.numTimesSelected * 100.00) / totalSelections
-              << "% " << entries[i].second.numProcessedRows;
+  for (int i = 0; i < maxEntriesLimit; i++) {
+    int idx = entries.size() - 1 - i;
```

Comment:
If entries.size() is 0 here, should we check that idx is non-negative before accessing entries?

Reply:
In this case, if entries.size() is zero then the maxEntriesLimit will be zero as well and the for loop will not run. Therefore we wont run into a negative idx